Cartfile.resolved

```diff
@@ -1,16 +1,15 @@
 binary "https://api.mapbox.com/downloads/v2/carthage/mapbox-common/MapboxCommon-ios.json" "4.0.0"
-binary "https://api.mapbox.com/downloads/v2/carthage/mobile-maps/mapbox-ios-sdk-dynamic.json" "6.1.0"
-binary "https://api.mapbox.com/downloads/v2/carthage/mobile-navigation-native/MapboxNavigationNative.json" "18.0.3"
+binary "https://api.mapbox.com/downloads/v2/carthage/mobile-maps/mapbox-ios-sdk-dynamic.json" "6.2.0"
+binary "https://api.mapbox.com/downloads/v2/carthage/mobile-navigation-native/MapboxNavigationNative.json" "18.0.4"
```

Comment:
These changes reflect what any developer would’ve obtained by default when running `carthage update` or `pod update` unless their Cartfile or Podfile happens to constrain the version further. The version numbers here reflect the maximum compatible version at the time of writing, not to be confused with the minimum compatible version defined in the Cartfile and podspecs. The maximum compatible version may change over time as these dependencies publish new minor and patch releases.

The only actual effect of this change is to the versions that I and others receive when running `carthage bootstrap` in this repository when setting up a development environment.